.github/workflows/build.yml

```diff
@@ -118,6 +118,13 @@ jobs:
         # --no-scan:      For public fork PRs, we won't attempt to publish the scan
         run: |
           ./gradlew --build-cache --info $SCAN_ARG check -x test
+          ./gradlew siteDocTar
```

Comment:
I think @chia7712's comment was to move the siteDocTar into this "Compile and validate" and add a subsequent step to verify the contents (e.g., "Check Site Docs"). Having two gradle commands in one step makes it harder to see what failed without digging into the logs.